end_to_end/signal_handler_test.go

```diff
@@ -11,6 +13,14 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+func CreateOutput(cmd *exec.Cmd) (*bufio.Scanner, strings.Builder) {
```

Comment:
What is this function supposed to do? Why it does return two different entities for an almost equal things - stdout and stderr?

Reply:
The main idea is to wait for the first line on stdout. To do this, we create a scanner on the stdout pipe. We also want to capture the output from stderr. Therefore, we create a stringbuilder to write there what we read through the scanner and also assign this stringbuilder to stderr so that it is written there automatically. This function is needed to reduce code duplication.

Reply:
So, this function creates output scanner and builder.  I suggest to use a name like createOutputStream() or createOutputScanner if you prefer to combine stdout and stderr. Note that it's used only from this file, no need to export it, so the name starts with lower 'c'.